terraform-kickstarter/main.tf

```diff
@@ -0,0 +1,535 @@
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third_party archives.
+
+   Copyright [2020] [© 2020 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE_2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+Security Hub Import Commands
+
+Run this to get state of SecurityHub
+
+terraform import aws_securityhub_account.securityhubresource 123456789012
+
+*/
+
+#TODO Move these to varaibles.tf file
+
+variable "enable_security_hub" {
+  description = "Enable AWS SecurityHub."
+  type        = bool
+  default     = true
+}
+
+variable "enable_security_hub_prowler_subscription" {
+  description = "Enable a Prowler Subscription."
+  type        = bool
+  default     = true
+}
+variable "prowler_cli_options" {
+  description = "Run Prowler With The Following Command"
+  type        = string
+  default     =  "-q -M json-asff -S -f us-east-1"
+}
+variable "prowler_schedule"{
+  description = "Run Prowler based on cron schedule"
+  default="cron(0 0 ? * * *)"
+  type=string
+
+}
+
+variable "codebuild_timeout" {
```

Comment:
should we separate the variables and data to other files?

Reply:
I have this as a TODO.  Thanks for the comment!

Reply:
Ok i was able to break out the various components.

Reply:
oh! nice! I saw the changes, much better now!